CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKFPSRenderable.java

```diff
@@ -325,41 +329,52 @@ private static int LoadShaders(CVKDevice cvkDevice) {
         int ret = VK_SUCCESS;
         
         try {
-            ByteBuffer vsBytes = LoadFileToDirectBuffer(CVKShaderPlaceHolder.class, "compiled/SimpleIcon.vs.spv");
+            vsBytes = LoadFileToDirectBuffer(CVKShaderPlaceHolder.class, "compiled/SimpleIcon.vs.spv");
```

Comment:
The shader load code should be unified somewhere with the md5 thing I mentioned above.  Good thing you added a shader utils class ;)

Reply:
Added to shader md5 task